src/global_legacy_module/4C_global_legacy_module.cpp

```diff
@@ -345,12 +345,8 @@ namespace
                     }),
                     parameter<bool>("SPECIAL"),
                 }),
-                selection<Solid::ResultTest::TestOp>("OP",
-                    {{"unknown", Solid::ResultTest::TestOp::unknown},
-                        {"sum", Solid::ResultTest::TestOp::sum},
-                        {"max", Solid::ResultTest::TestOp::max},
-                        {"min", Solid::ResultTest::TestOp::min}},
-                    {.default_value = Solid::ResultTest::TestOp::unknown}),
+                parameter<Solid::ResultTest::TestOp>(
+                    "OP", {.default_value = Solid::ResultTest::TestOp::unknown}),
```

Comment:
This diff nicely shows one of the benefits of magic enum: We can skip the manual mapping of strings to enums and are guaranteed to have consistent values.